<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>1042</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path of transformation</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startAgentButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>640</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Agent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seed (32 characters)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="walletKey">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>example: this_is_wallet1_password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wallet type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open policy agent url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genesisUrl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>http://192.168.0.5:9000/genesis</string>
   </property>
   <property name="placeholderText">
    <string>example: http://host.docker.internal:9000/genesis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1. Wallet (load an exsiting wallet or create a new wallet by starting the agent)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Optional Field</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genesis transaction url</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Mandatory Field</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.A Seed for a fixed public DID (Don't assert if you are loading a wallet containing a seed)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wallet name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transformationPath">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>../transformation</string>
   </property>
   <property name="placeholderText">
    <string>example: ../OPA/transformation.json</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Credential type</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="walletName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>example: wallet1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wallet password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="opaPolicyPath">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>../OPA</string>
   </property>
   <property name="placeholderText">
    <string>example: ../OPA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path of OPA policy files</string>
   </property>
  </widget>
  <widget class="QComboBox" name="credentialType">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>indy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>json-ld</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="walletType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>indy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>basic</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="opaUrl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>http://192.168.0.5:8181</string>
   </property>
   <property name="placeholderText">
    <string>example: http://host.docker.internal:8181</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="seed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>example: Zhang000000000000000000000000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="agent_name_label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>690</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Response</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="responseTextBrowser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>730</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9.16364pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.16364pt;&quot;&gt;Waiting for response.....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="acm">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>rbac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>abac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>capbac</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Access control model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Note: following arguments will be randomly generated if you don't assert them.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="loadingWallet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="text">
    <string>I am loading an exsiting wallet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agentOperationsButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>640</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agent Operations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="loadingWallet_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="text">
    <string>Connect to Access Control Engine</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>External host ip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="externalHost">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>example: http://host.docker.internal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
